Dashboard/app/js/lib/views/devices/assignments-list-view.jsx

```diff
@@ -64,7 +64,9 @@ FLOW.AssignmentsListView = FLOW.ReactComponentView.extend(observe({
   assignmentEdit(action, assignmentId) {
     switch (action) {
       case 'new': {
-        const newAssignment = FLOW.store.createRecord(FLOW.SurveyAssignment, {});
+        const newAssignment = FLOW.store.createRecord(FLOW.SurveyAssignment, {
+          name: Ember.String.loc('_new_question_please_change_name'),
```

Comment:
why question? is the string identical? if so maybe we should rename it something more meaningful?

Reply:
changed